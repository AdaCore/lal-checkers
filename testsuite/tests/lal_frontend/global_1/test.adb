procedure Test is
   X : Integer;

   procedure F is
   begin
      X := 12;
   end F;
begin
   F;
end Test;
