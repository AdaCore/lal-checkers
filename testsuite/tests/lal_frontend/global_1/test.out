Program:
  $stack#1 = Updated_1($stack#1, Input())
  ret0#2 = <SubpBody ["F"] test.adb:4:4-7:10>(Var_1($stack#1), Offset_2($stack#1))
  $stack#1 = CopyOffset($stack#1, Get_0(ret0#2))
  end:
